src/app/shared/state/theme/theme.reducer.ts

```diff
@@ -9,7 +10,9 @@ const initialState: Theme = {
 export default function reducer(state = initialState, action: ActionWithPayload<any>) {
   switch (action.type) {
     case ThemeActions.LOAD_THEME: {
-      return action.payload;
+      const newState = cloneDeep(state);
```

Comment:
Do we need cloneDeep here?

Reply:
in total, we could use the single property of Theme model but I preferred to make a clone of this becuse if we add something to our theme in future we wouldn't make a lot of time to make this correct again